starter-code/src/clue.js

```diff
@@ -1,14 +1,189 @@
+        occupation: "Socialité",
+        age: 36,
+        description: "She is from a wealthy family and uses her status and money to earn popularity",
+        image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+        color: "blue",
+    },
+
+    //mrMustard<br>
+    {
+        firstName: "Jack",
+        lastName: "Mustard",
+        occupation: "Retired Football player",
+        age: 62,
+        description: "He is a former football player who tries to get by on his former glory",
+        image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+        color: "yellow",
+    }
+];
 
 // Rooms Collection
-const roomsArray = [];
+const roomsArray = [{
+        name: "Dining Room"
+    },
+    {
+        name: "Conservatory"
+    },
+    {
+        name: "Kitchen"
+    },
+    {
+        name: "Study"
+    },
+    {
+        name: "Library"
+    },
+    {
+        name: "Billiard Room"
+    },
+    {
+        name: "Lounge"
+    },
+    {
+        name: "Ballroom"
+    },
+    {
+        name: "Hall"
+    },
+    {
+        name: "Spa"
+    },
+    {
+        name: "Living Room"
+    },
+    {
+        name: "Observatory"
+    },
+    {
+        name: "Theater"
+    },
+    {
+        name: "Guest House"
+    },
+    {
+        name: "Patio"
+    }
+];
+
 
 // Weapons Collection
-const weaponsArray = [];
+const weaponsArray = [{
+        name: "rope",
+        weight: 10
+    },
+    {
+        name: "knife",
+        weight: 8
+    },
+    {
+        name: "candlestick",
+        weight: 2
+    },
+    {
+        name: "dumbbell",
+        weight: 30
+    },
+    {
+        name: "poison",
+        weight: 2
+    },
+    {
+        name: "axe",
+        weight: 15
+    },
+    {
+        name: "bat",
+        weight: 13
+    },
+    {
+        name: "trophy",
+        weight: 25
+    },
+    {
+        name: "pistol",
+        weight: 20
+    }
+];
 
 // ITERATION 2
+/* 
+At the beginning of the game, players shuffle each of the card 
+stacks to create a combination of suspect, weapon and room. This will 
+be the mystery to solve.
+
+Random selector
+Declare a function named selectRandom to randomly select one element 
+from a card stack. The function should expect an array as an argument, 
+and should return a random element from the array.
+
+Create the mystery
+Declare a function named pickMystery that takes no arguments and 
+returns an object with three properties, suspect, weapon and room, 
+each holding as a value a card of that specific type. You can get a 
+random card of each type by calling selectRandom on each card stack.
+*/
+
+function selectRandom(theSuspect) {
+    let ourSuspect = Math.floor(Math.random() * theSuspect.length);
+    // console.log("ourSuspect",ourSuspect)
+    // console.log("mir",theSuspect[ourSuspect])
+    return theSuspect[ourSuspect].firstName
+}
+
+selectRandom(suspectsArray)
+console.log(selectRandom(suspectsArray))
+
```

Comment:
so far so good, please do the next iteration, it will help you to learn even more, and let me know if you need help